Ginger/Ginger/Actions/ActBrowserElementEditPage.xaml.cs

```diff
@@ -73,7 +84,7 @@ private ePlatformType GetActionPlatform()
         private void SetVisibleControlsForAction()
         {
             ResetView();
-            ePlatformType ActivityPlatform = GetActionPlatform();
+            ePlatformType ActivityPlatform = mAct.Platform;
 
```

Comment:
next time please use mAct.Platform directly where needed instead setting it into another parameter.
approving the PR anyway.